content/en/docs/guides/certificate_management/certificate-rotation.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Root Certificate Rotation"
+description: "Rotating the root certificate in OSM"
+type: docs
+weight: 10
+---
+
+# Root Certificate Rotation
+
+The Mesh's Root certificate is a long-lived certificate that is used by OSM to issue leaf certificates to services in the mesh thereby enabling mTLS. This document focuses on the steps to rotate a certificate, to learn more about OSM's certificates see [Certificate Provider Options](certificates.md).  
+
+The certificate can be rotated either through a [manual process](#manual-root-certificate-rotation) or with OSM v1.3 we have introduced a preview feature to help [automate rotation](#certificate-rotation-with-meshrootcertificate) and minimize downtime using a new API called the `MeshRootCertificate`.  
+
+### Manual Root Certificate Rotation
+
+#### Tresor
```

Comment:
May just be me but I got confused with the headings, maybe it could be like
Rotating with Tresor
Rotating with Other Certificate Providers
Rotating with MeshRootCertificate